action.yml

```diff
@@ -48,15 +51,47 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - uses: actions/checkout@v2
-    - run: |
+    - name: Checkout the fresh project
+      uses: actions/checkout@v2
+
+    - name: Configure git
+      run: |
         git config --global user.email "${{ inputs.actor-email }}"
         git config --global user.name "${{ inputs.actor-name }}"
-        npm install
       shell: 'bash'
-    - uses: actions/github-script@v5
+
+    - name: Version bump
+      id: version-bump
+      if: ${{ github.event_name == 'workflow_dispatch' }}
+      run: |
+        npm version --no-git-tag-version ${{ inputs.semver }}
+        echo "::set-output name=NPM_VERSION::$(echo $(node -p "require('./package.json').version"))"
```

Comment:
let's not call this output variable NPM_VERSION, it's confusing. PACKAGE_VERSION perhaps?